luxonis_train/core/trainer.py

```diff
@@ -56,7 +90,7 @@ def _upload_logs(self) -> None:
 
     def _trainer_fit(self, *args, **kwargs):
         try:
-            self.pl_trainer.fit(*args, **kwargs)
+            self.pl_trainer.fit(*args, ckpt_path=self.resume, **kwargs)
```

Comment:
Unimportant for this PR, but when we resume from checkpoint, should we consider also resuming the state of some hyperparameters? I'm not sure if that is already abstracted and/or a feature of pytorch lightning.

Reply:
Yes, we should probably store all states (optimizer, scheduler, ...) so we can truly continue with the training.

Reply:
Yeah, pytorch lightning does this automatically.